### PR TITLE
Update custom-entities to 1.19.2

### PR DESCRIPTION
**Pull request by Konicai**

### CODE REVIEW

File: core/src/main/java/org/geysermc/geyser/GeyserImpl.java [hunk 1]
```diff
@@ -178,24 +169,28 @@ private GeyserImpl(PlatformType platformType, GeyserBootstrap bootstrap) {
...
+
+        /* Initialize translators */
+        EntityDefinitions.init();
+        ItemTranslator.init();
+        MessageTranslator.init();
+        MinecraftLocale.init();
+
         /* Call Registry events */
-        Registries.callRegistryEvents();
+        Registries.callRegistryEvents(); // todo: before or after the above inits?
```
Comment (Konicai) [inline core/src/main/java/org/geysermc/geyser/GeyserImpl.java:193]: Just want to make sure this is seen. I assume its in the right position

**Comment by Konicai** [comment]
Comparing to feature/extensions: https://github.com/GeyserMC/Geyser/compare/feature/extensions...Konicai:Geyser:custom-entities-1.19.1

**Comment by TypicalShavonne** [comment]
is there a doc for creating the entity ? i wanted to test this

**Comment by Camotoy** [comment]
There is currently no documentation besides the Javadocs already in the code.

**Comment by TypicalShavonne** [comment]
which class is it ?

**Comment by Creeperz653** [comment]
Question does this have custom model support into this because I saw you merge feature and extensions  into this (ik this is stupid but I’m just askin)

**Comment by TypicalShavonne** [comment]
yes

**Comment by Creeperz653** [comment]
I’m moving to this branch then lol

**Comment by byquanton** [comment]
[core/src/main/java/org/geysermc/geyser/translator/protocol/java/entity/spawn/JavaAddEntityTranslator.java](https://github.com/GeyserMC/Geyser/commit/818baf466b888cd0399dced6b8856f57ab72c025#r92021001)

**Comment by Konicai** [comment]
Closed in favour of #3754

**Pull request closed without merging**